auth-api/src/auth_api/resources/org.py

```diff
@@ -393,6 +393,40 @@ def get(org_id):
         return response, status
 
 
+@cors_preflight('GET,OPTIONS')
+@API.route('/affiliation/<string:business_identifier>', methods=['GET', 'OPTIONS'])
+class OrgInfoByAffiliation(Resource):
+    @staticmethod
+    @TRACER.trace()
+    @cors.crossdomain(origin='*')
+    @_jwt.has_one_of_roles(
+        [Role.SYSTEM.value, Role.STAFF_VIEW_ACCOUNTS.value, Role.PUBLIC_USER.value])
+    def get(business_identifier):
+        """Search orgs by BusinessIdentifier and return org Name and UUID."""
+        org_search = OrgSearch(
```

Comment:
I'd recommend changing the OrgSearch dataclass to have optional fields?

Reply:
I added new service call, new call in model and updated the endpoint call. Let me know if it is better now. 

Thanks!